### PR TITLE
Add JsonUtil to reduce code duplication

### PR DESCRIPTION
**Pull request by kingthorin**
In follow-up to:
- https://github.com/zaproxy/zap-extensions/pull/1634#issuecomment-390972710
- https://github.com/zaproxy/zaproxy/pull/4677
- And IRC discussions.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/api/ApiUtil.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.api;
+
+import net.sf.json.JSONException;
+import net.sf.json.JSONSerializer;
+
+public class ApiUtil {
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply src/org/zaproxy/zap/extension/api/ApiUtil.java]: Ok I'll put it in the utils package but rename it. Sound good?

[the remark this answers is not in the document]
Comment (thc202) [reply src/org/zaproxy/zap/extension/api/ApiUtil.java]: Yeah.

File: src/org/zaproxy/zap/extension/api/ApiResponseSet.java [hunk 2]
```diff
@@ -49,18 +48,9 @@ public JSON toJSON() {
...
-					JSONSerializer.toJSON(value);
-					// Its valid JSON so escape
-					jo.put(val.getKey(), "'" + value + "'");
-				} catch (JSONException e) {
-					// Its not a valid JSON object so can add as is
-					jo.put(val.getKey(), value);
-				}
-			} else {
+				jo.put(val.getKey(), JsonUtil.getJsonFriendlyString((String) value));
+			} else
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/api/ApiResponseSet.java]: The braces could be kept ;)

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/api/ApiResponseSet.java]: Oops

**Comment by thc202** [comment]
Thanks!

**Pull request merged**